napari/_qt/qt_layerlist.py

```diff
@@ -139,15 +193,37 @@ def _ensure_visible(self, layer):
         self.ensureWidgetVisible(widget)
 
     def keyPressEvent(self, event):
+        """Ignores key press event.
```

Comment:
```suggestion
        """Ignore a key press event.
```

btw why does this exist, @sofroniewn?

Reply:
Nick answers this [in another comment](https://github.com/jni/napari/pull/5#issuecomment-589027107), he says:

> The `event.ignore` in the qt_layerlist allows the event to pass through a parent widget to its child widget without doing anything, otherwise the parent catches the event and it doesn't get passed on.